IPython/html/static/edit/js/main.js

```diff
@@ -29,22 +29,29 @@ require([
     ){
     page = new page.Page();
 
-    var base_url = utils.get_body_data('baseUrl');
-    var file_path = utils.get_body_data('filePath');
-    contents = new contents.Contents({base_url: base_url});
-    var config = new configmod.ConfigSection('edit', {base_url: base_url});
+    var common_options = {
+        ws_url : utils.get_body_data("wsUrl"),
+        base_url : utils.get_body_data("baseUrl"),
+        file_path : utils.get_body_data('filePath'),
+        notebook_path : utils.get_body_data("notebookPath"),
+        notebook_name : utils.get_body_data('notebookName')
+    };
+
+    // var base_url = utils.get_body_data('baseUrl');
+    // var file_path = utils.get_body_data('filePath');
+    var config = new configmod.ConfigSection('edit', common_options);
```

Comment:
(e.g. if we hadn't been passing common_options to Contents, we wouldn't have run into this confusion in the first place)